trace-utils/tests/snapshots/compare_v04_trace_meta_struct_snapshot_test.json

```diff
@@ -0,0 +1,68 @@
+[
```

Comment:
Do you think `meta_struct` should have its own snapshot test? Or should we incorporate it into the existing ones? I think integration tests should be separated by scenarios rather than the data.

Reply:
I chose that option because in order to fill the meta_struct I had to create a span instance rather than serializing from json which I find useful since serializing directly from our own representation could hide some errors but maybe I'm overthinking it a bit because the serialization process should be ok as long as the test agent verifies the payloads. WDYT? Do you see value in having a test that does the serialization from json?